.github/hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "Running ESLint & Karma"
+
+grunt lint
```

Comment:
grunt is for human use. Script with underlying commands: run `eslint ./src/js` or the `npm run lint` if you want js and less linting.